packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -388,6 +394,10 @@ class DragTarget<T> extends StatefulWidget {
   /// Called when an acceptable piece of data was dropped over this drag target.
   final DragTargetAccept<T> onAccept;
 
+  /// Called when a given piece of data being dragged over this target leaves
+  /// the target.
+  final DragTargetLeave<T> onLeave;
```

Comment:
We should probably add to the onWillAccept docs that that is called when a piece of data enters the target, and that that will be followed by either onAccept or onLeave.